files/build_templates/docker_image_ctl.j2

```diff
@@ -5,6 +5,31 @@ function getMountPoint()
     echo $1 | python -c "import sys, json, os; mnts = [x for x in json.load(sys.stdin)[0]['Mounts'] if x['Destination'] == '/usr/share/sonic/hwsku']; print '' if len(mnts) == 0 else os.path.basename(mnts[0]['Source'])" 2>/dev/null
 }
 
+function updateHostName()
+{
+    HOSTS=/etc/hosts
+    HOSTS_TMP=/etc/hosts.tmp
+
+    EXEC="docker exec -i {{docker_container_name}} bash -c"
+
+    NEW_HOSTNAME="$1"
+    HOSTNAME=`$EXEC "hostname"`
+    if ! [[ $HOSTNAME =~ ^[a-zA-Z0-9.\-]*$ ]]; then
+        HOSTNAME=`hostname`
+    fi
+
+    # copy HOSTS to HOSTS_TMP
+    $EXEC "cp $HOSTS $HOSTS_TMP"
+    # remove entry with hostname
+    $EXEC "sed -i \"/$HOSTNAME$/d\" $HOSTS_TMP"
+    # add entry with new hostname
+    $EXEC "echo -e \"127.0.0.1\t$NEW_HOSTNAME\" >> $HOSTS_TMP"
+
+    echo "Set hostname in {{docker_container_name}} container"
+    $EXEC "hostname '$NEW_HOSTNAME'"
+    $EXEC "cat $HOSTS_TMP > $HOSTS"
```

Comment:
Suggest to add `$EXEC "rm -f $HOSTS_TMP"`

Reply:
Added